app/javascript/dashboard/routes/dashboard/helpcenter/components/ArticleTable.vue

```diff
@@ -140,51 +147,8 @@ export default {
 };
 </script>
 <style lang="scss" scoped>
-.article-container {
-  @apply w-full;
-
-  .article-container--header {
-    @apply my-0 -mx-4 py-0 px-4 grid grid-cols-8 gap-4 border-b border-slate-100 dark:border-slate-700;
-
-    @media (max-width: 1024px) {
-      @apply grid-cols-7;
-    }
-
-    @media (max-width: 768px) {
-      @apply grid-cols-6;
-    }
-
-    &.draggable {
-      div.heading-item.heading-title {
-        @apply py-2 px-3.5;
-      }
-    }
-
-    div.heading-item {
-      @apply font-semibold capitalize text-sm text-right py-2 px-0 text-slate-700 dark:text-slate-100;
-
-      &.heading-title {
-        @apply text-left col-span-4;
-      }
-
-      @media (max-width: 1024px) {
-        &.heading-read-count {
-          @apply hidden;
-        }
-      }
-
-      @media (max-width: 768px) {
-        &.heading-read-count,
-        &.heading-last-edited {
-          @apply hidden;
-        }
-      }
-    }
-  }
-
-  .footer {
-    @apply p-0 border-0;
-  }
+.article-list-header-item {
+  @apply font-semibold capitalize text-sm py-2 px-0 text-slate-700 dark:text-slate-100;
```

Comment:
Let's apply this style directly instead of using `@apply` in case you feel this is too repetitive, we can create a new component

Reply:
Moved to inline